### ISSUE TITLE
[PageTitleBar] Storybook: collapsed status example has wrong text and icon color

### ISSUE THREAD

**Issue by mzuliani-ibm** [carbon-design-system/carbon-addons-iot-react#3346]
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

The storybook example for PageTitleBar - with status description and breadcrumbs

### To Reproduce

1. Go to [PageTitleBar - with status description and breadcrumbs](https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-pagetitlebar--with-status-description-and-crumbs)
2. Click on collapse description in the knobs
3. The status icon and text are unreadable on the dark tooltip

### Expected behavior

Icon and text style should be appropriate for the dark tooltip. 

### Environment/versions:

- OS: macOS
- Browser: chrome
- `carbon-addons-iot-react` version: [?]

### Additional context

![2022-02-24_16-49-28](https://user-images.githubusercontent.com/11927321/155615099-af8da8f8-a4d6-4112-9ca6-effd4c9a9e14.png)

![2022-02-24_16-49-57](https://user-images.githubusercontent.com/11927321/155615081-2ffc399f-d906-493c-8abe-15e2943ed7c2.png)
